docs/developing.md

```diff
@@ -0,0 +1,58 @@
+# Development guide
+
+This guide explains how to make changes to the `frontend-shared` package and use them with your local copy of the `client`.
```

Comment:
I suggest it might be clearer to replace `client` with a link to the client - ie. `[Hypothesis client](https://github.com/hypothesis/client)`